app/models/miq_event.rb

```diff
@@ -145,6 +145,7 @@ def self.raise_evm_job_event(target, options = {}, inputs = {}, q_options = {})
       :suffix => nil
     )
 
+    # note: this varies a little from compliance policy name
```

Comment:
suggestion to make the comment more searchable

```suggestion
    # NOTE: this varies a little from compliance_policy_name
```

Reply:
decided to remove these and leave the event base name / event policy name derivation alone.

Probably the best fix is to change the policies around the physical server and container image rather than introducing this concept.

Reply:
looks like one place used underscore and confused me. swapping it with downcase fixes most of the special cases